modules/ocf_kubernetes/manifests/master/loadbalancer.pp

```diff
@@ -2,6 +2,38 @@
   include ocf::firewall::allow_web
 
   $kubernetes_worker_nodes = lookup('kubernetes::worker_nodes')
+  $kubernetes_workers = $kubernetes_worker_nodes.map |$worker|
+    { [$worker, ldap_attr($worker, 'ipHostNumber')] }
```

Comment:
nit: Ruby usually keeps the `|$var|` and the `{` or `do` on the same line, so I think making this like:

```
  $kubernetes_workers = $kubernetes_worker_nodes.map |$worker| {
    [$worker, ldap_attr($worker, 'ipHostNumber')]
  }
```

would make it clearer to read and more idiomatic puppet.

Reply:
Fixed